src/utilities/util_printlog.c

```diff
@@ -12,33 +12,58 @@ static int
 usage(void)
 {
     static const char *options[] = {"-f", "output to the specified file", "-x",
-      "display key and value items in hexadecimal format", NULL, NULL};
+      "display key and value items in hexadecimal format", "-l", "start and stop the printlog",
+      NULL, NULL};
 
-    util_usage("printlog [-x] [-f output-file]", "options:", options);
+    util_usage("printlog [-x] [-f output-file] [-l start-file,start-offset,end-file,end-offset]",
```

Comment:
I think this should be "[-l start-file,start-offset]|[-l start-file,start-offset,end-file,end-offset]"